streams/src/main/java/org/apache/kafka/streams/kstream/ValueTransformerWithKey.java

```diff
@@ -75,14 +75,20 @@
     void init(final ProcessorContext context);
 
     /**
-     * Transform the given [key and ]value to a new value.
+     * Transform the given [key and] value to a new value.
      * Additionally, any {@link StateStore} that is {@link KStream#transformValues(ValueTransformerWithKeySupplier, String...)
      * attached} to this operator can be accessed and modified arbitrarily (cf.
      * {@link ProcessorContext#getStateStore(String)}).
      * <p>
-     * Note, that using {@link ProcessorContext#forward(Object, Object)} or
+     * Note that using {@link ProcessorContext#forward(Object, Object)} or
      * {@link ProcessorContext#forward(Object, Object, To)} is not allowed within {@code transform} and
      * will result in an {@link StreamsException exception}.
+     * <p>
+     * Note that if a {@code ValueTransformerWithKey} is used in a {@link KTable#transformValues(ValueTransformerWithKeySupplier, String...)}
+     * (or any other overload of {@code KTable#transformValues(...)}) operation,
+     * that the provided {@link ProcessorContext} from {@link #init(ProcessorContext)}
```

Comment:
is this a typo? 
if a 'xxx' is used in a 'yyy' operation, **that** the provided 'zzz' from 'aaa' doesn't guarantee....

Maybe replace `that` into `then`?